ironfish/src/chainProcessor.test.ts

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { BlockHeader, ChainProcessor } from '.'
+import { Assert } from './assert'
+import { createNodeTest } from './testUtilities'
+import { makeBlockAfter } from './testUtilities/helpers/blockchain'
+
+describe('ChainProcessor', () => {
+  const nodeTest = createNodeTest()
+
+  it('processes chain', async () => {
+    const { strategy, chain } = nodeTest
+    strategy.disableMiningReward()
+
+    // G -> A1
```

Comment:
I think that should be:
// G -> A1 -> A2 -> A3
       -> B1  -> B2

Reply:
Yeah, updated the comment.